widgetry/src/drawing.rs

```diff
@@ -161,7 +161,7 @@ impl<'a> GfxCtx<'a> {
     pub fn draw_mouse_tooltip(&mut self, txt: Text) {
         // Add some padding
         let pad = 5.0;
-
+        let txt = txt.default_fg(self.style.text_tooltip_color);
```

Comment:
This is the one place I changed to update default tooltip text color across the app